merfi/backends/rpm_sign.py

```diff
@@ -57,18 +60,19 @@ def sign(self):
             raise RuntimeError('specify a --key for signing')
         logger.info('Starting path collection, looking for files to sign')
         repos = RepoCollector(self.path)
```

Comment:
what if we could tell `RepoCollector` that we want to collect DEBs or RPMs? Or maybe have one RPMRepoCollector and one DEBRepoCollector?